tests/unit/writers/test_skoscodelistwriter.py

```diff
@@ -46,24 +45,27 @@ def test_code_list_data_mapping():
     data = writer._get_code_list_data()
     actual_column_names = {str(c) for c in data.columns}
     assert {
+        "Uri Identifier",
         "Label",
         "Notation",
-        "Parent Notation",
+        "Parent Uri Identifier",
         "Sort Priority",
         "Description",
     } == actual_column_names
 
     first_concept: Dict[str, Any] = data.iloc[[0]].to_dict("records")[0]
+    assert first_concept["Uri Identifier"] == "1st-20-concept"
     assert first_concept["Label"] == "First Concept"
-    assert first_concept["Notation"] == "1st-concept"
-    assert first_concept.get("Parent Notation") is None
+    assert first_concept["Notation"] == "1st-20%concept"
```

Comment:
Why has this changed?

Reply:
This is because I have changed the `basic_code_list` in line 15 to include `20%` which will be detected by `uri_safe()` so we can unit test this aspect.